### ISSUE TITLE
GPU Meter confused by "0 MHz" core clock

### ISSUE THREAD

**Issue by johny-mnemonic** [SilverAzide/Gadgets#18]
I have a new AMD RX 6800 and it pretty often reports 0 MHz core clock while idle, doing nothing.
It seems, like GPU Meter is confused by this, as whenever GPU clock goes to zero, it stops showing any values and reports it is not configured.

I had to switch it to use "GPU Clock (Effective)" sensor instead of "GPU Clock", to workaround it, as that one seems to somehow average the clock, so it never goes to zero.

I am using new 6.0.0 version of gadgets with Rainmeter 4.4.0.3438 and HWInfo 7.0.2-4430.

**Comment by SilverAzide** [comment]
Thanks for the feedback!  I didn't expect the clock speed to ever go all the way to zero!  Please make the following change to the `CalcIsHWiNFOConfigured` measure (this will restore the logic to match the previous version and should never be zero).
``` ini
[CalcIsHWiNFOConfigured]
Measure=Registry
RegHKey=HKEY_CURRENT_USER
RegKey=SOFTWARE\HWiNFO64\VSB
RegValue=ValueRaw[#HWiNFO_GPU[#GpuIndex]_Temp]
RegExpSubstitute=1
; if zero, set to No; otherwise, set to Yes
Substitute="^0$":"No","^\d+\.\d+$":"Yes","^\d+$":"Yes"
```
I'll incorporate this change in the next release.

**Comment by johny-mnemonic** [comment]
Now you risk, that some OC guys with freezers will have issues 😃

**Comment by johny-mnemonic** [comment]
I am sorry to say that I have found another bug. Memory clock is showing the same value as GPU clock:
```
[MeterGPUMemClock]
Meter=String
MeterStyle=StyleTextInfo
StringAlign=Right
FontColor=#PaletteColor1#
Text="[MeasureGPUClock:0] MHz"
```

**Comment by SilverAzide** [comment]
Ugh... Yeah, looks like I managed to goof that up when I converted everything to the new HWiNFO 7.x measures.  Thanks for the feedback!

As far as the freezer folks, that's why I changed to use the clock instead of the temp.  I didn't think the clock would ever go to zero, but obviously I forgot about sleeping/idling.  I just need some sensor that should never be zero, but unfortunately there aren't any others that I know of if you include all the goofy Intel GPUs, old ATI GPUs, etc.  I'll just have to hope no one cools their rig to 0 C (or less), LOL (CPU Meter uses the same logic).
